VimeoNetworking/VimeoNetworking/Models/VIMVideo.m

```diff
@@ -437,7 +437,16 @@ - (BOOL)isRatedMature
 
 - (BOOL)isDRMProtected
 {
-    return self.playRepresentation.drmFiles.fairPlayFile != nil;
+    VIMInteraction *buyInteraction = [self interactionWithName:VIMInteractionNameBuy];
+    BOOL isBuyDRMProtected = buyInteraction.drm;
+    
+    VIMInteraction *rentInteraction = [self interactionWithName:VIMInteractionNameRent];
+    BOOL isRentDRMProtected = rentInteraction.drm;
+    
+    VIMInteraction *subscribeInteraction = [self interactionWithName:VIMInteractionNameSubscribe];
+    BOOL isSubscribeDRMProtected = subscribeInteraction.drm;
+    
+    return isBuyDRMProtected || isRentDRMProtected || isSubscribeDRMProtected;
```

Comment:
Is this still necessary now that we've enabled the TV app on the API side to pull the fairplay files?